build/pipeline-releasebuild.yaml

```diff
@@ -21,299 +21,38 @@ variables:
   BuildConfiguration: 'release'
 
 steps:
```

Comment:
doesn't the release build need the bootstrap template?

Reply:
Yes, it's there on line 28.  The release build does the code analysis first to look at the source.